content/apply.js

```diff
@@ -462,13 +462,10 @@ const APPLY = (() => {
         transition: none !important;
       }
     `)
-      .then(() => {
-        // repaint
-        // eslint-disable-next-line no-unused-expressions
-        document.documentElement.offsetWidth;
-        removeStyle({id: 0});
-        document.documentElement.classList.remove(className);
-      });
+    setTimeout(() => {
```

Comment:
`setTimeout` should be put in `.then(() => { ... })`. The current code only works if `setTimeout` is always executed after `applySections` is complete.

Reply:
Done. I just copied the code  we were using previously, but the goal here is to wait as long as possible anyway,